modules/ephemeral-storage-using-a-sharedconfimap-object-in-a-pod.adoc

```diff
@@ -0,0 +1,81 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="ephemeral-storage-using-a-sharedconfigmap-object-in-a-pod_{context}"]
```

Comment:
There is a typo in the name of the file: `sharedconfimap` instead of `sharedconfigmap` (missing g)